CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ current changes on git with [previous release tags][git_tag_comparison].
 
 ### Added
 
+- [Added run criteria example][7388]
```

Comment:
This semi-automatically generated each release; can you revert your changes to this file?

Reply:
wait a second